### ISSUE TITLE
Have to press reset everytime for a new compression else an error screws up the app

### ISSUE THREAD

**Issue by sriksm19** [ghosh/Crimp#2]
First off, thanks for the app
Problem:
Cant drag new files onto the window when a set of optimized files are already present. If done, then the app throws an error. Have to reset all the time before adding new files. (lazy me :P)

What should happen:
Drag and drop should automagically reset and compress. 

Error:
![image](https://user-images.githubusercontent.com/1223117/31381504-6189473c-add2-11e7-903b-1548503b867e.png)
![image](https://user-images.githubusercontent.com/1223117/31381519-6cb3f7ec-add2-11e7-85b6-908da25e9f98.png)
![image](https://user-images.githubusercontent.com/1223117/31381526-72e84870-add2-11e7-9b5a-676c8a81544c.png)

**Comment by ghosh** [comment]
@sriksm19 Yup, that's something I plan on fixing sometime soon. Thanks for reporting it.
